core/cap-0023.md

```diff
@@ -344,6 +341,20 @@ The behavior of `CreateClaimableBalanceOp` is as follows:
     - `asset` as specified in the operation
     - `amount` as specified in the operation
     - `reserve` equal to `claimants.size() * baseReserve`
+
+[^id-arithmatic]: ```
+    OperationID = (
+      sourceAccount ||
+      seqNum ||
+      opNum
+    )
+    balanceID.v0() = hex(sha256(OperationID))
+    ```
+    - `OperationID`: a union within the new `type` `ENVELOPE_TYPE_OP_ID`
```

Comment:
Looks like this was missing in the original description as well, but the union being hashed is the `HashIDPreimage` where the type is `ENVELOPE_TYPE_OP_ID`. Do you mind adding that in the description?

Reply:
About half of the [community](https://github.com/stellar/js-stellar-sdk/issues/1021) SDKs use `HashIdPreimage` rather than `HashIDPreimage` (see @kalepail). Do you have any source on the preferred [stylization](https://github.com/stellar/stellar-docs/issues/799)? I'm partial to your caps version, but wanted to maintain [style deference](https://chatgpt.com/share/f79edef6-c4cc-4a23-abfd-2e810dfbd9dc) to @overcat.

Reply:
Oh interesting. I just copied it directly from the [xdr file](https://github.com/stellar/stellar-xdr/blob/78ef9860777bd53e75a8ce8b978131cade26b321/Stellar-transaction.x#L678).